Godeps/_workspace/src/k8s.io/kubernetes/pkg/securitycontextconstraints/provider.go

```diff
@@ -228,20 +241,7 @@ func (s *simpleProvider) ValidateContainerSecurityContext(pod *api.Pod, containe
 		allErrs = append(allErrs, fielderrors.NewFieldInvalid("privileged", *sc.Privileged, "Privileged containers are not allowed"))
 	}
 
-	if sc.Capabilities != nil && len(sc.Capabilities.Add) > 0 {
-		for _, cap := range sc.Capabilities.Add {
-			found := false
-			for _, allowedCap := range s.scc.AllowedCapabilities {
-				if cap == allowedCap {
```

Comment:
I'll just toUpper them.  It works fine for the docker values but the docker documentation has them written in caps too.  These values are inherently container runtime dependent though.  For instance you don't say you want CAP_SYS_ADMIN when adding the cap in docker you just use SYS_ADMIN.  Another example - docker has a special case for the ALL value.